Client/Internal/ApiClient.cs

```diff
@@ -108,10 +110,17 @@ private void InitToken()
                     var cookies = authResponse.Cookies.ToList();
 
                     if (cookies.Count == 1)
-                        _sessionToken = cookies
-                            .First(cookie => cookie.Name.ToString().Equals("session"))
+                    {
```

Comment:
Couldn't this be simplified to remember all cookies (or key/value pairs), which could be all later sent in the `Signout` method?

Reply:
That is very true, but I would also say that based on the code, it still seems that InfluxDB will only ever return 1 cookie; it's just that the name is not known at compile time anymore. 

So if you would prefer to open up that assumption and tolerate 1+ cookies sent, I can make the change and respond back.

Reply:
@sranka simple and straightforward as usual 👍

Reply:
@sbhenderson feel free to use @sranka's suggestion. It will be better for future maintenance.

Reply:
I have made the change in line with @sranka's suggestion to store all names/values for using in future requests.